include/definitions.h

```diff
@@ -38,12 +38,14 @@
 
 #define PCF8574_NUM_DEVICES 4
 #define PCF8574_NUM_PORTS 8
-#define I2C_ADDRESS_PCF8574_IOExt0 0x20 // TODO: set jumpers accordingly
-#define I2C_ADDRESS_PCF8574_IOExt1 0x21 // TODO: set jumpers accordingly
-#define I2C_ADDRESS_PCF8574_IOExt2 0x22 // TODO: set jumpers accordingly
-#define I2C_ADDRESS_PCF8574_IOExt3 0x23 // TODO: set jumpers accordingly
+
+#define IOExtPINCOUNT PCF8574_NUM_DEVICES*PCF8574_NUM_PORTS
+#define I2C_ADDRESS_PCF8574_IOExt0 0x20
+#define I2C_ADDRESS_PCF8574_IOExt1 0x21
+#define I2C_ADDRESS_PCF8574_IOExt2 0x22
+#define I2C_ADDRESS_PCF8574_IOExt3 0x23
 #define I2C_INTERRUPT_PIN_PCF8574 33
-#define USE_LEGACY_PIN_READ
+
```

Comment:
Did you remove the code that needed this macro? (Then, it is ok to delete it here, otherwise not)

Reply:
yes I did